### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Ibtisam34**
- [ ]  This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I had created making use of React and Redux.

I selected an API that provides numeric data about a topic that I liked and then build we web around it. The web appill has several pages:

1. one page with a list of items that could be filtered by some parameters.

2. one page for the item details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,149 @@
...
+
+Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+ESlint (is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs).
+🛠 Built With Tech Stack Client Server Database
+
+# 🚀 Live Demo
+Please use the following link to access the website:
+[live demo](https://riyana5678.netlify.app/)
+[live presentation](https://www.loom.com/share/fddfc6960af443308443f38ba92144b6)
```
Comment (fobadara) [inline README.md]: [OPTIONAL] Kindly give your live presentation a heading and put it on a separate line. I almost made the absence of a video link a required change

Comment (fobadara) [inline README.md]: [REQUIRED] Kindly reduce the video to 5 minutes. The requirements are to present your project, make it a maximum of  5 minutes and show us a piece of code you are proud of. There was no need to show us the bulk of your code

File: README.md [hunk 2]
```diff
@@ -1 +1,149 @@
...
+
+# Deployment
+You can deploy this project using:
+
+Please run the following to deploy
+npm run deploy
+
+
+
+👥 Authors
```
Comment (fobadara) [inline README.md]: [OPTIONAL] Kindly make. `authors` singular

File: README.md [hunk 3]
```diff
@@ -1 +1,149 @@
...
+
+⭐️ Show your support
+Write a message to encourage readers to support your project
+
+If you like this project...
+
+🙏 Acknowledgments
+Microverse for providing the Design and guideline for this project.
+(back to top)
+
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). You need to include the links also

File: README.md [hunk 4]
```diff
@@ -1 +1,149 @@
...
+Which index.html file should I open to make some changes in src folder or dist folder?
+
+Please make change only to Html, CSS and JS files in the src directory
+(back to top)
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```
Comment (fobadara) [inline README.md]: [OPTIONAL] Some of. your subheadings are squeezed together like authors, future features, contribution, show your support, license and faq. Make them bold also for example:
```
## Faq
```
The two `#` and a space will make it a subheading or a single `#` for a main heading

File: src/pages/homepage/weathercountries.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+          <Country
+            key={country.iso3}
+            name={country.name}
+            states={country.states}
+            iso3={country.iso3}
+          />
+        ))}
+      </div>
+    </div>
+  );
```
Comment (fobadara) [inline src/pages/homepage/weathercountries.js]: [REQUIRED] One of the aims of this project is to teach you to follow client requirements. Kindly ensure your app design is close to Nelson's template in terms. of typography and layout
image



[OPTIONAL] I think adding pagination and a search bar to your app will make it more user friendly

File: src/pages/weatherstate/weatherstates.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+      <div className="states padding">
+        {countryState.states.map((state) => (
+          <State
+            name={state.name}
+            key={state.name}
+            code={state.state_code}
+          />
+        ))}
+      </div>
```
Comment (fobadara) [inline src/pages/weatherstate/weatherstates.js]: [REQUIRED] One of the aims of this project is to teach you to follow client requirements. Kindly ensure your app design is close to Nelson's template in terms. of typography and layout

image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Ibtisam34, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/homepage/weathercountries.js (hunk 5 above)
Comment (Ibtisam34) [reply src/pages/homepage/weathercountries.js]: https://github.com/fobadara sir here Iam not getting could you plz join huddle to tell me further

File: src/pages/homepage/weathercountries.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
```
Comment (fobadara) [inline src/pages/homepage/weathercountries.js:38]: [REQUIRED] Some of your texts are going out of its box

File: src/pages/homepage/weathercountries.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        <p className="choose-country">Choose a country</p>
+        <div className="country-list-header">
+          <h2 className="country-list-title">Countries</h2>
+        </div>
+      </div>
+      <div className="country-card-container">
+        {Object.values(countries).map((country) => (
+          <Country
+            key={country.iso3}
+            name={country.name}
```
Comment (fobadara) [inline src/pages/homepage/weathercountries.js:31]: [REQUIRED] Try to focus more on the mobile screen size Kindly make sure that you have two columns for items without space at the top, bottom, left or right just like the design. The boxes should be touching each other

image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Ibtisam34, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some haven't been addressed yet.


 ## Highlights
✅ Linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hello @Ibtisam34 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**